rpmrc.in

```diff
@@ -80,6 +80,9 @@ optflags: armv6hl -O2 -g -march=armv6 -mfloat-abi=hard -mfpu=vfp
 optflags: armv7l -O2 -g -march=armv7
 optflags: armv7hl -O2 -g -march=armv7-a -mfloat-abi=hard -mfpu=vfpv3-d16
 optflags: armv7hnl -O2 -g -march=armv7-a -mfloat-abi=hard -mfpu=neon
+optflags: armv8l -O2 -g -march=armv8-a
+optflags: armv8hl -O2 -g -march=armv8-a -mfloat-abi=hard -mfpu=neon
```

Comment:
`armv8hl` should not have neon.

Reply:
Same here, since NEON is mandatory for ARMv8 (that's why there is no -mfpu= switch in the aarch64-* compilers), armv8hl and armv8hnl are the same and -mfpu=neon will do the right thing.

I don't see any use case for the combination of armv8-a and -mfpu=something other than neon, except in the softfloat case (armv8l) if you need to use a binary-only library that still doesn't support hardfloat (Android...)

Reply:
Technically speaking, NEON is mandatory for the 64-bit ISA, but as far as I'm aware, that is not the case for the 32-bit ISA. AArch32 v8 still allows NEON to be omitted.